breakings/2.3.0/dialog.yml

```diff
@@ -0,0 +1,11 @@
+- scope: 'component'
+  name: 'el-dialog'
+  type: 'props'
+  version: '2.3.0'
+  commit_hash: '412c79c'
```

Comment:
Maybe we should add this file when the API really removed.

Reply:
Ok, I'll remove it later and resolve the conflict.